utils/paraswap/swap-helper.js

```diff
@@ -0,0 +1,282 @@
+const { assert, expect } = require("chai");
```

Comment:
paraswap helper was named `sell-helper` because Paraswap codebase distinguishes a `buy` (acquiring a fixed amount of tokens by sending a variable amount of tokens) from a `sell` (sending a fixed amount of tokens to acquire a variable amount of tokens). Our app only uses the `sell` function for now. So you could merge the content of `swap-helper` (which related to `sell` functions) into `sell-helper`.

Reply:
I agree, so can we remove the `buy-helper`?

Also whenever a file reaches 200-300 lines I think it ok to split over multiple files.

Reply:
Yes, `buy-helper` can be removed.

Agree with splitting large files but here it's used as a library of related methods so I think it's fine to keep it as one.